### ISSUE TITLE
Display source of selected text/object

### ISSUE THREAD

**Issue by SafaAlfulaij** [QupZilla/qupzilla#1589]
Sometimes I want to view just the source of a text or quarter of the page.
Or maybe the source from the beginning of an object till the end of it.

**Comment by ff2000** [comment]
Right click on that element and select "Inspect Element".

**Comment by SafaAlfulaij** [comment]
Yes, But I want to view the source code to copy it (for example). Inspector isn't that easy to copy from it.
